engine/src/conversion/analysis/struct_accessors.rs

```diff
@@ -0,0 +1,195 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use std::collections::HashSet;
```

Comment:
Please avoid using `HashSet` or `HashMap` and instead use some collection which has predictable iteration order. (I tend to use `use indexmap::set::IndexSet as HashSet`) so that we have deterministic behavior.